src/Common/Models/TaskModel.cs

```diff
@@ -10,20 +10,39 @@ public class TaskModel : BaseModel
         [JsonPropertyName("Name")]
         public override string Name { get; set; }
 
-        [JsonPropertyName("ResourceLock")]
+        [JsonPropertyName("ReplanActionPlannerLocked")]
+        public bool ReplanActionPlannerLocked { get; set; } //Dont change actions in action sequence replan
```

Comment:
Here are a lot of the properties responsible solely for the replan. Maybe it will be better to have the replan config as a separate model? What do you think?